Automattic/wp-calypso
# VideoPress Onboarding: Add new config for Videomaker trial and modify a few steps

This PR gets us started with some tweaks to the VideoPress onboarding flow to use a trial version of the Videomaker theme.

It adds a new config for the trial so that we won't change the current flow until we're ready.



Fixes https://github.com/Automattic/greenhouse/issues/1897

## Testing Instructions



* Ensure you run `yarn` to build the new config files.
* Load http://calypso.localhost:3000/setup/videopress locally.
* Choose `Showcase your work` and create a new site.
* The flow won't be perfect, but the payment step should be skipped and you should end up with a Videomaker theme on the site (the `pub/videomaker` one).
* Test the calypso.live build and ensure it still takes you to the checkout page (feature should only be enabled for `development`)

## Pre-merge Checklist
